Source/Examples/README.md

```diff
@@ -1,49 +1,73 @@
-   - Finally, the new configuration is exported to disk.
-   
-### Using a previously exported configuration (part2):
-   - Using the configuration file (exported at the end of the previous example) an Instrument Addon Custom Device is created with the same configuration.
-   - A Response section is added to the message added in the previous example. 
-   - A Data Channel is also added.
-   - Finally, the Response is configured to use the Data Channel Refnum.
-   
-### RMX410x Configuration:
-   - A new Instrument Addon is created in the target System Definition File, with an RS-xxx protocol type.
-   - Configuration, Control and Feedback groups are created.
-   - Messages configured with MetaData and Data channels are created.
-   - The Messages for controlling and getting feedback from the RMX410x device, and Channels that control the output state and voltage levels are configured.
-   - [RMX410x User manual](https://www.ni.com/pdf/manuals/377396b.pdf)
-   
-## **Instructions on how to run the examples:**
-
-1. Open the Instrument Addon Scripting API Examples.lvproj
-2. Open the desired VI and modify the message type, settings, config file paths as desired.
-3. Run the VI
-
-**Note:** It is strongly advised to operate the modifications mentioned in step number 2, as the logic in the VIs is for demonstration purposes only. You may need another wait type for your Response, other Channel types, padding may not make sense in your use case, etc.
\ No newline at end of file
+### Basic Example
+
+`Basic Example.vi` configures an Instrument Addon with a single message, both Command and Response, and exports the configuration to a file on disk.
+
+The VI executes the following steps:
+   1. Creates a new TCP Protocol object and uses the protocol as an input for a new Instrument Addon Custom Device.
+   1. Updates the name of the add-on.
+   1. Creates a message section and a new message.
+   1. Adds and configures the Command and Response sections.
+
+
+   **Note:** The **settings** input for the **Configure Response** uses a known value, 42, as the channel refnum for the linked channel. This channel is composed of a variable name and a channel refnum.
+
+### Alter the IP Address of a Protocol
+
+`Alter the IP Address of a protocol.vi` alters the IP Address of a protocol while leaving other properties unchanged.
+
+The VI executes the following steps:
+   1. Uses the input refnum to fetch the protocol of the Instrument Addon Custom Device.
+   1. Determines if the protocol is **TCP** or **AK TCP**.
+   1. Alters the IP Address and inputs the new value to the appropriate protocol factory.
+
+The factory produces a protocol object that is used as an input for the **Set Protocol** node.
+
+### Setting a new Protocol, Regardless of Previous Configuration
+
+### RMX410x Configuration
+
```

Comment:
Could I get a description for what this VI is supposed to do? I think that would help

Reply:
Creates an Instrument Addon custom device and adds the Messages and Channels used to Control and get Feedback from the NI RMX410x Rackmount Programmable DC Power Supply.